src/Diagnostics.ModelsAndUtils/Models/GeoMaster/AzureSupportCenterInsight.cs

```diff
@@ -0,0 +1,164 @@
+            where TResource : IResource
+        {
+
+            // Logic in line before would have to be updated if we added more resource types. 
+            // TODO: May make sense to have resource type enums contain the resource type string
+            var resourceTypeString = context.Resource.ResourceType == ResourceType.App ? "sites" : "hostingEnvironments";
+            var applensPath = $"subscriptions/{context.Resource.SubscriptionId}/resourceGroups/{context.Resource.ResourceGroup}/{resourceTypeString}/{context.Resource.Name}/detectors/{detector.Id}";
+
+            var recommendedAction = GetTextObjectFromData("recommended action", insight.Body);
+            var description = GetTextObjectFromData("description", insight.Body);
+            var customerReadyContent = GetTextObjectFromData("customer ready content", insight.Body);
+
+            var supportCenterInsight = new AzureSupportCenterInsight()
+            {
+                Id = GetDetectorGuid(detector.Id),
+                Title = insight.Message,
+                ImportanceLevel = ImportanceLevel.SomeLevel,
+                InsightFriendlyName = insight.Message,
+                IssueCategory = detector.Name.ToUpper(),
+                IssueSubCategory = detector.Name,
+                Description = description,
+                Links = new List<Link>()
+                    {
+                        new Link()
+                        {
+                            Type = 0,
+                            Text = "Applens Link",
+                            Uri = $"https://applens.azurewebsites.net/{applensPath}"
+                        }
+                    },
+                RecommendedAction = recommendedAction == null ? null : 
+                    new RecommendedAction()
+                    {
+                        Id = new Guid(),
+                        Text = recommendedAction
+                    },
+                CustomerReadyContent = customerReadyContent == null ? null :
+                    new CustomerReadyContent()
+                    {
+                        Text = customerReadyContent
+                    },
+                ConfidenceLevel = "High",
+                Scope = "ResourceLevel"
+            };
+
+            return supportCenterInsight;
+        }
+
+        private static Guid GetDetectorGuid(string detector)
+        {
+            try
+            {
+                Encoding utf8 = Encoding.UTF8;
+                int count = utf8.GetByteCount(detector);
+                byte[] bytes = new byte[count > 16 ? count : 16];
+                Encoding.UTF8.GetBytes(detector, 0, detector.Length, bytes, 0);
+                // Guid only takes byte array of length 16
+                return new Guid(bytes.Take(16).ToArray());
+            }
+            catch(Exception ex)
+            {
+                Console.WriteLine(ex.ToString());
```

Comment:
:) . Remove this.

Reply:
Oh yes this is another piece left over from testing, good catch